onmt/translate/decode_strategy.py

```diff
@@ -1,4 +1,38 @@
 import torch
+import onmt
+
+
+def get_decode_strategy(beam_size, batch_size, bos, eos, pad, device,
```

Comment:
It does not seem like this function has much value at this time. I suggest building the decoding strategy object directly in `translate_batch` as suggested.